backend/app/schemas/schedule.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum
+from uuid import UUID
+from datetime import datetime, timedelta
+from typing import List, Optional
+from app.schemas.time_block import TimeBlockBase, TimeBlockId, TimeBlockFull
+from pydantic import BaseModel, ConfigDict
+
+
+
+class ScheduleState(str, Enum):
+    PENDING_PARTICIPANT = "PENDING_PARTICIPANT_RESPONSE"
+    PENDING_VOLUNTEER = "PENDING_VOLUNTEER_RESPONSE"
```

Comment:
nit: keep these in the same order as in the DB, just in case someone uses this as a reference for the ordering.